datamol/align.py

```diff
@@ -40,7 +39,7 @@ def template_align(
     use_depiction: bool = True,
     remove_confs: bool = True,
     auto_select_coord_gen: bool = False,
-):
+) -> Union[Mol, None]:
```

Comment:
Thanks.

The recommended way is `Optional[Mol]`. Can you switch to that?

Reply:
done @hadim